stable_baselines3/common/env_checker.py

```diff
@@ -285,34 +285,22 @@ def _check_spaces(env: gym.Env) -> None:
 
 
 # Check render cannot be covered by CI
-def _check_render(env: gym.Env, warn: bool = True, headless: bool = False) -> None:  # pragma: no cover
+def _check_render(env: gym.Env) -> None:  # pragma: no cover
     """
-    Check the declared render modes and the `render()`/`close()`
+    Check the instantiated render mode (if any) by calling the `render()`/`close()`
     method of the environment.
 
     :param env: The environment to check
     :param warn: Whether to output additional warnings
     :param headless: Whether to disable render modes
         that require a graphical interface. False by default.
     """
-    render_modes = env.metadata.get("render.modes")
-    if render_modes is None:
-        if warn:
-            warnings.warn(
-                "No render modes was declared in the environment "
-                " (env.metadata['render.modes'] is None or not defined), "
-                "you may have trouble when calling `.render()`"
-            )
-
-    else:
-        # Don't check render mode that require a
-        # graphical interface (useful for CI)
-        if headless and "human" in render_modes:
-            render_modes.remove("human")
-        # Check all declared render modes
-        for render_mode in render_modes:
-            env.render(mode=render_mode)
-        env.close()
+    # render_modes = env.metadata.get("render.modes")
+    # TODO: if we want to check all render modes,
+    # we need to initialize new environments so the class should be passed as argument.
+    if env.render_mode:
```

Comment:
why did you remove the available render mode declaration?
they are not longer declared in gym 0.26?

Reply:
They are still there ( I think), but unless we can instantiate new class instances we cannot test any other mode than the one from the current instance.

Reply:
> ny other mode than the one from the current instance.

yes, I'm more talking about what we use to test with the warning (check if any render mode is declared and output a warning otherwise)

Reply:
ah yes, my bad. I was too hasty apparently, will bring that check back